src/Ocean/OceanProblems/SimpleBoxProblem.jl

```diff
@@ -1,435 +0,0 @@
-module OceanProblems
```

Comment:
Does it make sense to rename this file instead of deleting?

Reply:
I think I did do this --- I created a new eponymous `OceanProblems.jl` to correspond to the `OceanProblems` module (which I understand is convention --- though it is not followed in `ClimateMachine`?) I moved the "simple box" code into it's own file, `simple_box_problem.jl`.